Wasted/Pages/Place/PageManager.cs

```diff
@@ -7,12 +7,12 @@ public static class PageManager
     {
         public static void PushClientPage(ContentPage page)
         {
-            page.Navigation.PushAsync(new MainPage());
+            page.Navigation.PushAsync(new Client.MainPage());
         }
 
         public static void PushPlacePage(ContentPage page)
         {
-            page.Navigation.PushAsync(new PlaceMainPage());
+            page.Navigation.PushAsync(new MainPage());
```

Comment:
What was the purpose of renaming this class?

Reply:
Just to fit the style of Client type user, where we have "MainPage" and not "ClientMainPage". Also the old name can be confusing when there are such files as "FoodPlacePage".